lbz/resource.py

```diff
@@ -154,13 +159,10 @@ def __init__(
     ):
         # TODO: adjust the rest of the arguments in the near future too.
         super().__init__(event)
-        cors_headers = cors_headers or []
```

Comment:
This line was pretty important as the code below assumes that `cors_headers` is always a list.

Reply:
and it is - default ensures that.